src/server/scripts/Spells/spell_hunter.cpp

```diff
@@ -3143,6 +3144,61 @@ class PlayerScript_black_arrow : public PlayerScript
     }
 };
 
+/// Bursting Shot --  186387
+class spell_bursting_shot : public SpellScript
+{
+    PrepareSpellScript(spell_bursting_shot);
+
+    enum eSpells
+    {
+        AURA_SHOTING = 224729
```

Comment:
You should put this spell in the top-file enum in format SPELL_HUNTER_AURA_SHOOTING